deploy/.env.example

```diff
@@ -0,0 +1,6 @@
+# Required: Your private key
```

Comment:
Maybe use the extension .dist instead

Reply:
It's convention which is already adopted by some templates 

> 

.dist files are often configuration files which do not contain the real-world deploy-specific parameters (e.g. Database Passwords, etc.), and are there to help you get started with the application/framework faster.